packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_model.dart

```diff
@@ -25,6 +25,7 @@ class UpdateOtherSoftwareModel extends ChangeNotifier {
     notifyListeners();
   }
 
+  // ignore: avoid_positional_boolean_parameters
```

Comment:
proposing to ignore this one because the signature is a perfect pair to CheckboxListTile.onChanged